dandi/support/bids/utils.py

```diff
@@ -0,0 +1,200 @@
+"""
```

Comment:
why these files are added back in this PR? (likely some rebase went a bit "dirty")

Reply:
Hmmm... Yesterday I

-> implemented `etelemetry`
-> synched `catalystneuro:master` to `dandi-cli:master`
-> merged `catalystneuro:master` into this branch

will look into this to see what went wrong

Reply:
OK whatever caused it it must have happened before the `etelemetry` change. It's been manually removed from this branch now